lib/libthr/thread/Makefile.inc

```diff
@@ -59,3 +58,6 @@ SRCS+= \
 	thr_symbols.c \
 	thr_umtx.c \
 	thr_yield.c
+
+SIMPLE_PRINTF_PREFIX=_thread
+.include "${SRCTOP}/lib/libsimple_printf/Makefile.inc"
```

Comment:
Maybe have the changes in this file depend on ${MACHINE_ABI:Mpurecap} so we don't break non mips and riscv targets?  I guess you'd also need to rename `_thread_printf`...

Reply:
It should work for all targets but I haven't verified it. The assembly version of write is only used if available. Otherwise it uses __sys_write

Reply:
In that case this seems fine.